File: src/SOP8/Egovframework/Base/base-account-ibll/src/main/java/egovframework/base/account/ibll/request/RequestUpdateUserInfo.java
package egovframework.base.account.ibll.request;

public class RequestUpdateUserInfo {

    private int userNo = -1;
    private String nickName = null;
    private Integer grade = null;
    private String memo = null;

    public int getUserNo() {
        return userNo;
    }

    public void setUserNo(int userNo) {
        this.userNo = userNo;
    }

    public String getNickName() {
        return nickName;
    }

    public void setNickName(String nickName) {
        this.nickName = nickName;
    }

    public Integer getGrade() {
        return grade;
    }

    public void setGrade(Integer grade) {
        this.grade = grade;
    }

    public String getMemo() {
        return memo;
    }

    public void setMemo(String memo) {
        this.memo = memo;
    }
}

File: src/SOP8/Base/Account/Base.Account.IBLL/Request/RequestUpdateUserInfo.cs
namespace Base.Account.IBLL.Request
{
    public class RequestUpdateUserInfo
    {
        private int m_nUserNo = -1;
        private string m_strNickName = null;
        private int? m_grade = null;
        private string m_strMemo = null;

        public int UserNo
        {
            get { return m_nUserNo; }
            set { m_nUserNo = value; }
        }

        public string NickName
        {
            get { return m_strNickName; }
            set { m_strNickName = value; }
        }

        public int? Grade
        {
            get { return m_grade; }
            set { m_grade = value; }
        }

        public string Memo
        {
            get { return m_strMemo; }
            set { m_strMemo = value; }
        }
    }
}
